README.md

```diff
@@ -1044,3 +1044,120 @@ class Pirate {
 
 }
 ```
+
+## 5. File Organization
+
+### 5.1 UIView + UIViewController Subclasses
+
+Guidelines:
+
+* **5.1.1** Files should include properties first, then init methods, then methods
+
+* **5.1.2** Property Grouping and Ordering
+    1. Prefer using closures to init and configure properties to separate `setupFooProperty` methods
+    2. Properties should be grouped first with Public, Internal, and then Private groups.
+
+* **5.1.3** Method Grouping and Ordering
+    1. Init Methods
+    2. Internal Methods
+    3. Overrides
+        If the class is overriding methods from a parent class, those methods should be grouped indicating which
+        parent the override belongs to. i.e. // MARK: UIView or // MARK: UIResponder for a UIView Subclass.
+    4. Private Methods  
+
+Example:
+
+```swift
+// Copyright © 2018 Helix OpCo. All rights reserved.
+
+import UIKit
+
+class ExampleView: UIView {
+
+    // MARK: Internal Properties
+
+    // MARK: Private Properties
+
+    // Init and configure subview that only requires initializer call
+    private let label = HelixLabel(textStyle: EmbeddedAppTextStyle.cellHeader)
+
+    // Init and configure subview that does not require self to configure.
+    private let imageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFill
+        return imageView
+    }()
+
+    // Init and configure view that requires self to configure.
+    private lazy var textField: UITextField = {
+        let textField: UITextField
+        textField.delegate = self
+        return textField
+    }()
+
+    // MARK: Init
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        setupSubviews()
+        setupConstraints()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: Internal Methods
+
+    func configure(viewModel: ViewModelType) {
+        
+    }
+
+    // MARK: UIView Overrides
```

Comment:
You know, I actually had it like that about five minutes before pushing this PR and changed it last minute. I think that yeah it's more natural to have it in the order you've stated here. My rationale for changing it was that maybe it makes more sense to have Internal methods closer to the top since as someone who is using a class, that is what you're most interested in finding when you open up a file, but actually I do think I like what you'e laid out here better. I'll update.